### DISCUSSION TITLE
upload to cloud

### DISCUSSION THREAD

**Discussion by sec13b** [projectdiscovery/nuclei#5746] [Q&A] [answered]
How i  upload  the results to cloud ?

I use : 
~~~~~~~~~~~~~~~~~~~~~~
nuclei  -l local.txt -t vulnerabilities/ -t cves/ -t exposures/ -t exposed-panels/ -t network/ -t miscellaneous/ -t misconfiguration/ -stats -bs 100 -c 50 -severity critical,high -o targets.txt
~~~~~~~~~~~~~~~~~~~~~~

**Comment by ehsandeep** [comment, marked as answer]
See https://docs.projectdiscovery.io/tools/nuclei/running#nuclei-result-dashboard

**Reply by sec13b** [reply]
-cloud-upload
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
nuclei -cloud-upload  -l local.txt -t vulnerabilities/ -t cves/ -t exposures/ -t exposed-panels/ -t network/ -t miscellaneous/ -t misconfiguration/ -stats -bs 100 -c 50 -severity critical,high -o targets.txt 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

where i add the api key?

**Reply by ehsandeep** [reply]
```
nuclei -h auth

-auth  configure projectdiscovery cloud (pdcp) api key
```

**Comment by sec13b** [comment]
can be any cloud , like dropbox , etc ? or is just cloud.projectdiscovery.io

**Reply by ehsandeep** [reply]
Yes, it's just https://cloud.projectdiscovery.io, for other platforms, you can upload local results manually.

**Comment by sec13b** [comment]
OK, 
I use nuclei -auth  configure projectdiscovery cloud 
I entered the API key .
I run :
nuclei -cloud-upload  -l local.txt -t vulnerabilities/ -t cves/ -t exposures/ -t exposed-panels/ -t network/ -t miscellaneous/ -t misconfiguration/ -stats -bs 100 -c 50 -severity critical,high -o targets.txt 

The output file with  found vulnerabilities is not on https://cloud.projectdiscovery.io/
